setup.py

```diff
@@ -17,6 +17,7 @@
     packages=find_packages(),
     include_package_data=True,
     install_requires=[
+        'Pillow>=5.4.0,<=7.0.0',
```

Comment:
Just noting that we should probably bump the version to 1.2 (rather than staying at 1.1.x) for the next release since Pillow is a relatively big dependency to add.

Reply:
Sounds good! If this is approved, I can commit the update to `setup.py` to bump the version number. Is there anywhere else that this needs to be changed?

Reply:
^^ @zwolf ?

Reply:
That'll be handled in a separate version release PR. If this feature needs the new version, update it here and I'll handle the new version release when this merges.